Anastasiya Santalova/hw1.css

```diff
@@ -0,0 +1,257 @@
+body {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+
+  min-width: 320px;
+  min-height: 320px;
+
+  margin: 0;
+  padding: 40px 20px;
+
+  font-size: 15pt;
+  font-family: "Avenir Next", monospace;
+  font-weight: 600;
+  text-transform: uppercase;
+  text-align: center;
+  color: #ffffff;
+
+  background-image: url(img/main-background.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center center;
+}
+
+.widget {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  
+  display: flex;
+  flex-flow: row wrap;
+  align-content: center;
+  justify-content: center;
+
+  overflow: auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 830px;
+  align-content: center;
+  justify-content: center;
+}
+
+.weather {
+  display: flex;
+  align-content: center;
+  align-items: center;
+
+  width: 274px;
+  height: 274px;
+  min-width: 250px;
+}
+
+.weather-skyblue {
+  background-color: #06b3db;
+}
+
+.weather-green {
+  background-color: #4ed486;
+}
+
+.weather-blue {
+  background-color: #6139f6;
+}
+
+.weather-orange {
+  background-color: #e3b63d;
+}
+
+.weather-pink {
+  background-color: #dc3d66;
+}
+
+.weather-purple {
+  background-color: #db51e2;
+}
+
+.weather__block_visible {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+
+  box-sizing: border-box;
+
+  width: 100%;
+  height: 100%;
+  padding-top: 20%;
+  padding-bottom: 15%;
+}
+
+.weather__image_large {
+  width: 110px;
+}
+
+.weather__city {
+  margin-bottom: 10px;
+}
+
+/* Invisible elements before hover */
+
+.weather__block_invisible {
+  display: none;
+  position: relative;
+}
+
+.weather__period-info {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+
+  padding: 0 30px;
+
+  width: 100%;
+  height: 100%;
+}
+
+.weather__background_transparency {
```

Comment:
I'd call it `transparent` since modifier name is rather adjective than noun like `invisible` above.

Reply:
Yes, you right. Fixed :)